llvm/include/llvm/BinaryFormat/SFrame.h

```diff
@@ -0,0 +1,165 @@
+//===-- llvm/BinaryFormat/SFrame.h ---SFrame Data Structures ----*- C++ -*-===//
```

Comment:
```suggestion
//===-- llvm/BinaryFormat/SFrame.h ---SFrame Data Structures --------------===//
```

Reply:
Note that per the latest coding standard, the file name is not needed in the header: https://llvm.org/docs/CodingStandards.html#file-headers